django/template/base.py

```diff
@@ -841,12 +840,12 @@ def __repr__(self):
     def render(self, context):
         try:
             output = self.filter_expression.resolve(context)
+            return render_value_in_context(output, context)
         except UnicodeDecodeError:
             # Unicode conversion can fail sometimes for reasons out of our
             # control (e.g. exception rendering). In that case, we fail
             # quietly.
             return ''
-        return render_value_in_context(output, context)
```

Comment:
As long as you can't crash the debug page, you're fine. I don't have a strong opinion one way or another. I just wanted to make sure it wasn't an inadventent change.